CHANGELOG.md

```diff
@@ -29,7 +29,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - The tool integrated `oegenerate` in this release.
 - SGX enclaves created using OE SDK can now be debugged using `oelldb`.
   `oelldb` is a python based extension for LLDB that supports debugging SGX enclaves. lldb-7 or above is required.
-
+- In Linux, 0-base enclave creation is enabled.
+  - Developers can create an enclave with base address 0x0 by either defining SGX_ENCLAVE_ZERO_BASE (before including headers) or
+    using the oesign tool config file option 'CreateZeoBasedEnclave'.
+  - Optionally, the enclave image start address can be provided by assigning the value to SGX_ENCLAVE_START_ADDRESS or via
+  config file option 'StartAddress'.
+  - tests/SampleAppZeroBase provides an example of 0-base enclave creation.
```

Comment:
Ideally, the example should be part of samples. Test one under tests is for testing purpose.